common-lib/src/main/scala/com/gu/mediaservice/lib/elasticsearch/Mappings.scala

```diff
@@ -123,6 +123,57 @@ object Mappings {
       "filename" -> nonAnalyzedString
     )
 
+  val usageReference =
+    nonDynamicObj(
+      "type" -> nonAnalyzedString,
+      "uri"  -> nonAnalyzedString,
+      "name" -> sStemmerAnalysedString
```

Comment:
OK :+1:. Is it worth adding to the API examples for completeness?

Or it could be done in the API (between ES and returning the JSON) so all clients get a nice representation but we don't denormalise something which is derived and harder to change later? Up to you.